File: Begginers/Python/soma_de_pares_consecutivos_1159.py
while True:
      valor = int(input())
      
      if valor == 0: break
  
      if valor % 2 == 0:
         total = 0
         for i in range(valor, valor + 9, 2):
             total += i
      else:
           total = 0
           for i in range(valor + 1, valor + 10, 2):
               total += i

      print(total)

File: Begginers/C++/soma_de_pares_consecutivos_1159.cpp
#include <stdio.h>

int main() {
 
    int valor, total, i;
    
    while (1) {
        scanf("%d", &valor);
        
        if (valor == 0) {
            break;
        }

        total = 0; 

        if (valor % 2 == 0) {
            for (i = valor; i <= valor + 9; i += 2) {
                total += i;
            }
        }
        else {
            for (i = valor + 1; i <= valor + 10; i += 2) {
                total += i;
            }
        }
        
        printf("%d\n", total);
    }
 
    return 0;
}
